GHZGame/ReferenceImplementation.qs

```diff
@@ -84,19 +82,18 @@ namespace Quantum.Kata.GHZGame {
 
     // Task 2.3. Play the GHZ game using the quantum strategy
     operation PlayQuantumGHZ_Reference (strategies : (Qubit => Bool)[]) : Bool[] {
-        mutable abc = new Bool[3];
 
         using (qs = Qubit[3]) {
             CreateEntangledTriple_Reference(qs);
-
+            
+            mutable abc = new Bool[3];
             for (i in 0..2) {
-                set abc[i] = strategies[i](qs[i]);
+                set abc w/= i <- strategies[i](qs[i]);
```

Comment:
@cgranade I think this was on of the places where I was looking for a library function to do it: apply i-th element of one array to i-th element of another and put the result in i-th element of the answer.

Reply:
@cgranade If such a library function/operation does not exist, that would be a good one to add the the M.Q.Array.